docker/shibboleth-sp-nginx/Dockerfile

```diff
@@ -1,17 +1,15 @@
 # Build stage
-FROM debian:buster-slim AS build
-
-ENV NGINX_VERSION=1.18.0-2~buster
```

Comment:
Wait just curious why do we no longer need this lol

Reply:
Honestly bc I don't know what the most recent version of `nginx` in that repository is and don't see why we need to pin to a specific Nginx version if the docker image is basically never rebuilt anyway. Seems like it will just make it harder to update in the future.

Maybe there's some reason why it's important to pin the version of nginx like that?